fsharp-backend/src/LibExecution/ProgramTypes.fs

```diff
@@ -975,25 +349,27 @@ type Op =
   | SetExpr of tlid * id * Expr
   | TLSavepoint of tlid
   | DeleteFunction of tlid
-  | CreateDBMigration of tlid * id * id * (string * id * string * id) list
-  | AddDBColToDBMigration of tlid * id * id
-  | SetDBColNameInDBMigration of tlid * id * string
-  | SetDBColTypeInDBMigration of tlid * id * string
-  | AbandonDBMigration of tlid
-  | DeleteColInDBMigration of tlid * id
```

Comment:
note to self: I'm not immediately sure why these could be removed - will check history and follow up here.